RELEASE.md

```diff
@@ -68,11 +50,7 @@
     set +e
     ```
 
-* push to local github
-* merge (this step and the above one can be skipped if you just do it on the the repo's release branch itself)
-* git tag (your version number)
-* git push --tags
-* get a clean release (as in, fresh clone)
-* npm publish
+* `npm publish`
+* `git push --follow-tags`
```

Comment:
Being pedantic, but I would imagine we want this line and the line above to be swapped. So, instead

```
git push --follow-tags
npm publish
```